data/campaigns/Northern_Rebirth/scenarios/13a_Showdown.cfg

```diff
@@ -822,7 +822,7 @@
         # Intro
         [message]
             id=Tallin
-            message= _ "Rakshas! You have evaded me for the last time. Now come out and die like a man!"
+            message= _ "Rakshas! You have escaped me for the last time. Now come out and die like a man!"
```

Comment:
Evaded is fine because you may or may not have forced him to retreat before.

Reply:
Oh yeah, I forgot, branching story.